VideoLocker/src/test/java/org/edx/mobile/test/http/UserProfilePresenterTest.java

```diff
@@ -0,0 +1,71 @@
+package org.edx.mobile.test.http;
+
+import org.edx.mobile.module.analytics.ISegment;
+import org.edx.mobile.module.prefs.UserPrefs;
+import org.edx.mobile.profiles.UserProfileImageViewModel;
+import org.edx.mobile.profiles.UserProfileInteractor;
+import org.edx.mobile.profiles.UserProfilePresenter;
+import org.edx.mobile.profiles.UserProfileViewModel;
+import org.edx.mobile.test.PresenterTest;
+import org.edx.mobile.util.observer.CachingObservable;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class UserProfilePresenterTest extends PresenterTest<UserProfilePresenter, UserProfilePresenter.ViewInterface> {
+
+    static final String PROFILE_USERNAME = "john_doe";
+
+    @Mock
+    UserProfileInteractor userProfileInteractor;
+    @Mock
+    UserPrefs userPrefs;
+    @Mock
+    ISegment segment;
+
+    CachingObservable<UserProfileViewModel> accountObservable;
+
+    CachingObservable<UserProfileImageViewModel> photoObservable;
```

Comment:
The local variables should be `private` or `protected`, I guess. This applies to the other test classes too.